Documentation/operations/upgrade.rst

```diff
@@ -378,6 +381,12 @@ Deprecated Options
 
 * ``bpf-compile-debug``: This option does not have any effect since 1.10
   and is planned to be removed in 1.11.
+* ``k8s-force-json-patch``: This option does not have any effect for
+  environments running Kubernetes >= 1.13. Marked for removal in Cilium v1.11.
+* ``masquerade``: With the introduction of IPv6 masquerading this option has
+  been deprecated in favor of ``enable-ipv4-masquerade`` and is planned to
+  be removed in 1.11. For 1.10 release this option will have the same effect as
+  ``enable-ipv4-masquerade`` where both options must not be used simultaneously.
```

Comment:
Ah, I see. I guess the question is whether we should have a separate "deprecated" vs. "removed" section. I don't know if we've had a discussion somewhere about this but it seems reasonable to me. :+1:

Reply:
Yeah, there was one item each under both sections even though they had the same behaviour(deprecated in upcoming and removed in future releases). So I removed `Deprecated Option` section.